### ISSUE TITLE
JAWS does not announce aria-describedby on role="region"

### ISSUE THREAD

**Issue by kabsas** [FreedomScientific/VFO-standards-support#709]
### Summary
JAWS does not announce aria-describedby on role="region"

Steps to reproduce:
1. Launch JAWS
2. Go to https://codepen.io/yououhan123/pen/bGxPBGG 
3. Navigate to button 2

### Expected result
JAWS announces "This is a region region, this is a description" followed by the announcement of the focused button

### Actual result
JAWS announces "This is a region region" followed by the announcement of the focused button. The description is not announced.

#### JAWS version and build number
2023.2302.15
 
#### Operating System and version
Windows 10 Enterprise 19044.2604

#### Browser and version:
Chrome 111
Edge 111
Firefox 110

**Comment by JAWS-test** [comment]
I have a hard time calling this a bug.
Question 1 would be: What do other screen readers do at this point?
Question 2 would be: Where in the spec does it say that when you enter a region, its label and description must be output?

My experience is that descriptions are usually only output at the element that receives focus. 

I would rather advocate that the specifications clarify more precisely when a description should be output

**Comment by stevefaulkner** [comment]
@BrettLewisVispero can reproduce, is there a reason why JAWS does not announce descriptions on regions?
